mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -206,12 +208,18 @@ class KMeans private (
     this
   }
 
+  def run(data: RDD[Vector]): KMeansModel = {
```

Comment:
This is the public method, so it needs to have the documentation and Since tag.  The private version does not.